packages/github-actions/actions/prepare-node/README.md

```diff
@@ -24,9 +24,9 @@ jobs:
         uses: actions/checkout@v3
```

Comment:
Could you clarify why we're not updating all instances of `actions/checkout` to `v4`? Even if it's not required would it not be better to use the same across all actions?

Reply:
Thanks for catching this. They should have been updated as well. Addressed in https://github.com/woocommerce/grow/pull/112/commits/d71c2c1ce2ebd74efb5554ce3467455a4da282c9.